.github/workflows/post-promote-production.yml

```diff
@@ -0,0 +1,28 @@
+name: post-promote-production
+
+on:
+  repository_dispatch:
+    types:
+      - post-promote-production
+      - post-promote-production::*
+
+permissions: {}
+
+jobs:
+  start-registry-check:
+    name: Start Registry Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate GitHub App Token
+        id: token
+        uses: actions/create-github-app-token@df432ceedc7162793a195dd1713ff69aefc7379e # v2.0.6
+        with:
+          app-id: ${{ secrets.APP_ID }}
+          private-key: ${{ secrets.APP_PEM }}
+
+      - name: Initiate Workflow
+        shell: bash
+        env:
+          GH_TOKEN: ${{ steps.token.outputs.token }}
+        run: |
+          gh workflow run --repo ${{ github.repository }} --ref main slack-test.yml
```

Comment:
is `slack-test.yml` defined in workflows?

Reply:
holdover from my test repo 🙃. I'll get that corrected